arkouda/strings.py

```diff
@@ -1936,7 +1936,7 @@ def to_parquet(
         - Any file extension can be used.The file I/O does not rely on the extension to
```

Comment:
wow sphinx really is picky-- way above this line in `strip` around 650ish. i think the parameters need to be on a new line to display correctly

Reply:
done